environment/aws/main.tf

```diff
@@ -39,7 +39,7 @@ data "aws_security_group" "main" {
 # This is the machine(s) that will run Couchbase Server
 resource "aws_instance" "couchbaseserver" {
     count = var.server_count
-    ami = "ami-05576a079321f21f8"
+    ami = "ami-0cae6d6fe6048ca2c"
     instance_type = "m5.xlarge"
```

Comment:
I don't think you have to make these changes now, but I think that you can/should think about using graviton instances for aws. If we are maxing the cpu, the instance is undersized.

on us-east1 for 16GB / 4 CPUs (which also is probably oversized for these use cases):

(cheapest)
m6g.xlarge .150/hr
m7g.xlarge .154/hr
m5.xlarge .19/hr

it isn't worth hyper optimzing this but I don't see a reason not to use arm64 images since we are never maxing cpu on SG or CBS